src/main/webapp/app/core/sentry/sentry.error-handler.ts

```diff
@@ -31,7 +32,24 @@ export class SentryErrorHandler extends ErrorHandler {
             dsn: profileInfo.sentry.dsn,
             release: VERSION,
             environment: this.environment,
-            integrations: [new ArtemisDeduplicate()],
+            integrations: (integrations) => {
+                integrations.push(new ArtemisDeduplicate());
+                if (this.environment !== 'local') {
+                    integrations.push(
+                        new BrowserTracing({
+                            routingInstrumentation: instrumentAngularRouting,
+                            beforeNavigate: (context) => {
+                                return {
+                                    ...context,
+                                    name: location.pathname.replace(/\/[a-f0-9]{32}/g, '/<hash>').replace(/\/\d+/g, '/<digits>'),
+                                };
+                            },
+                        }),
+                    );
+                }
+                return integrations;
+            },
+            tracesSampleRate: this.environment !== 'prod' ? 1.0 : 0.2,
```

Comment:
In getTracesSampleRace() (that you should rename...) 'local' has a smaple rate of 0.0, here it also has 0.2
Is that intended?

Reply:
For the client, the `BrowserTracing` integration is only added for non-local environments. :)